pkg/machine/hyperv/machine.go

```diff
@@ -513,7 +516,15 @@ func (m *HyperVMachine) Stop(name string, opts machine.StopOptions) error {
 	if vm.State() != hypervctl.Enabled {
 		return hypervctl.ErrMachineStateInvalid
 	}
+
+	defer func() {
+		if err := machine.CleanupGVProxy(m.GvProxyPid); err != nil {
+			logrus.Error(err)
+		}
```

Comment:
I think we need to clean up the pidfile here since gvproxy doesn't clean that up on exit